nservicebus/azure-service-bus/retries.md

```diff
@@ -5,36 +5,47 @@ tags:
 
-In order to implement Immediate Retries, NServiceBus maintains an internal counter to track the processing attempts by the endpoint instance for a specific message. If the number of attempts exceeds the `MaxRetries` setting, then the message will be forwarded to the error queue.
 
-Each endpoint's instance maintains its own internal counter for retries. If an endpoint is scaled out then the message can be retried a few times by each instance, before one of them decides to send the message to the error queue. So in effect the global retry count, as seen by the Azure Service Bus entity, will in the worst case scenario be equal to `MaxRetries * number of instances`.
+### Excessive prefetching and MaxDeliveryCount 
 
-In Azure Service Bus Versions 6 and below the default value for `MaxDeliveryCount` is 6. In Azure Service Bus Versions 7 and higher it has been changed to 10, based on the assumption that by default there are 2 instances of each endpoint and `MaxRetries` has the default value of 5.
+When a message is received for processing, the transport will also [prefetch](/nservicebus/azure-service-bus/configuration/full.md#controlling-connectivity-message-receivers) additional messages. "If processing a message takes too long time, then the broker will consider that a failed processing attempt and increase `DeliveryCount` counter on the messages w/o processing attempt taking place.  
 
+Endpoints with high `PrefetchCount` that take long time to process an incoming message can cause prefetched message to be sent to the Delayed Retries or the error queue without being retried the desired number of times. To prevent this issue, lower the `PrefetchCount`.
 
-### Elastic Scale
+See [Message Receivers configuration](/nservicebus/azure-service-bus/configuration/full.md#controlling-connectivity-message-receivers) for more details.
 
-The `MaxDeliveryCount` setting becomes a property on the Azure Service Bus entity. It can't be easily modified after the entity has been created. At the same time the number of instances can be changed just by dragging a slider in the Azure administration portal.
 
-This becomes a problem when the number of instances exceeds what was originally planned. The messages can be dead lettered when the total retry value exceeds the value configured in `MaxDeliveryCount`, which was adjusted for a smaller number of instances.
+## Retries and dead letter queues
 
-One way to solve this problem is to configure the entity to forward dead letter messages to the error queue:
+A valid message failing processing will be retried and eventually end up in the error queue. If a message is poisonous or cannot be moved to an error queue, it will be moved by the broker to a dead letter queue. Each endpoint has its own DLQ. Monitoring multiple DLQs might be challenging. One way to solve this problem is to configure the entity to forward dead letter messages to the error queue:
```

Comment:
>A valid message failing processing will be retried and eventually end up in the error queue. 

How about 

if a message could not be successfully processed even after retries, it will eventually end up in the error queue.